content/applications/finance/expenses.rst

```diff
@@ -79,88 +94,112 @@ Manually create a new expense
+     agrees to pay for the expenses associated with it (such as travel, hotel, meals, etc.). All
+     expenses tied to that meeting would indicate the sales order for the custom garden (which also
+     references the customer) as the :guilabel:`Customer to Reinvoice`.
+
+- :guilabel:`Analytic Distribution`: Select the account(s) the expense should be written against
+  from the drop-down menu for either :guilabel:`Projects`, :guilabel:`Departments`, or both.
+  Multiple accounts can be listed for each category if needed. Adjust the percentage for each
+  analytic account by typing in the percentage value next to the account.
+- :guilabel:`Company`: If multiple companies are set up, select the company this expense should be
+  filed for from the drop-down menu. The current company will automatically populate this field.
 - :guilabel:`Notes...`: If any notes are needed in order to clarify the expense, enter them in the
   notes field.
 
-Once all the fields have been filled out, click :guilabel:`Save`.
-
 .. image:: expenses/expense-filled-in.png
    :align: center
    :alt: A filled in expense form for a client lunch.
 
 Attach a receipt
 ~~~~~~~~~~~~~~~~
 
-After the expense is saved, the next step is to attach a receipt. A new :guilabel:`Attach Receipt`
-button appears after the entry is saved, beneath the former :guilabel:`Save` button (which turns
-into an :guilabel:`Edit` button).
+After the expense is created, the next step is to attach a receipt. Click the :guilabel:`Attach
+Receipt` button, and a file explorer appears. Navigate to the receipt to be attached, and click
+:guilabel:`Open`. The new receipt is recorded in the chatter, and the number of receipts will appear
+next to the :guilabel:`📎 (paperclip)` icon beneath the expense form. More than one receipt can be
+attached to an individual expense, as needed. The number of receipts attached to the expense will be
+noted on the paperclip icon.
 
-.. image:: expenses/save-receipt.png
+.. image:: expenses/receipt-icon.png
    :align: center
-   :alt: Attach a receipt after saving the record.
+   :alt: Attach a receipt and it appears in the chatter.
 
-Click the new :guilabel:`Attach Receipt` button, and a file explorer appears. Navigate to the
-receipt to be attached, and click :guilabel:`Open`. A new :guilabel:`Receipts` smart button appears
-at the top, and the new receipt is recorded in the chatter. More than one receipt can be attached to
-an individual expense, as needed. The number of receipts attached to the expense will be noted on
-the smart button.
+Create new expenses from a scanned receipt
+------------------------------------------
 
-.. image:: expenses/receipt-smartbutton.png
+Rather than manually inputting all of the information for an expense, expenses can be created by
+scanning a PDF receipt.
+
+First, in the main *Expenses* app dashboard view (this view can also be accessed from
```

Comment:
this might get a gui bc it's the name of a dashboard, no?
```suggestion
First, in the main :guilabel:`Expenses` app dashboard view (this view can also be accessed from
```